deployment/ansible/roles/model-my-watershed.geotrellis/tasks/compile.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Compile the scala code
+  shell: "sbt 'project services' compile assembly"
```

Comment:
I usually try to avoid using `shell` unless some type of shell expansion is required. Was there a reason not to use `command`?